build.sbt

```diff
@@ -21,6 +22,19 @@ lazy val scalametaRoot = Project(
 ) settings (
   sharedSettings,
   unidocSettings,
+  // needs to be a command since scripted requires publishing across scala-versions
+  commands += Command.command("scripted") { state =>
+    "such publishLocal" ::
+    "very scalahostSbt/scripted" ::
+    state
+  },
+  commands += Command.command("testAll") { state =>
```

Comment:
How about we rename `testAll` to something different, e.g. `ci`, so it doesn't make an impression of something that just runs all tests aggregated across projects?